docs/platforms/apple/common/configuration/options.mdx

```diff
@@ -89,6 +89,16 @@ Grouping in Sentry is different for events with stack traces and without. As a r
 
 </ConfigKey>
 
+<ConfigKey name="enable-sigterm-reporting">
+
+When enabled, the SDK reports SIGTERM signals to Sentry.
+
+It's crucial for developers to understand that the OS sends a SIGTERM to their app as a prelude to a graceful shutdown, before resorting to a SIGKILL. This SIGKILL, which your app can't catch or ignore, is a direct order to terminate your app's process immediately. Developers should be aware that their app can receive a SIGTERM in various scenarios, such as  CPU or disk overuse, watchdog terminations, or when the OS updates your app.
+
+_(New in [sentry-cocoa version 8.27.0](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#8270))_
```

Comment:
```suggestion
```

I'd move this up so that it's clear which option it's for (otherwise could look like it's for the option after this)